src/components/utils/index.js

```diff
@@ -30,18 +30,29 @@ let isIOS = (/iPad|iPhone|iPod/.test(navigator.platform || '') || (navigator.pla
 export const isDesktop = !(/Android|webOS|BB10|BlackBerry|IEMobile|Opera Mini|Mobile|mobile/i.test(navigator.userAgent || '')) && !isIOS
 
 // To detect hybrid desktop/mobile devices which have a rear facing camera such as the Surface
-export async function isHybrid() {
+export async function isHybrid(facingMode) {
   return isDesktop &&
     navigator.platform === 'Win32' &&
     await navigator.mediaDevices.getUserMedia(
       {
         video: {
-          facingMode: {
-            exact: "environment"
-          }
+          facingMode
         }
       }
-    ).then(() => true).catch(() => false);
+    ).then(async (mediaStream) => {
+      const devices = mediaStream.getTracks();
+      const matches = ['back', 'rear', 'world'];
+      let device = devices.find(d => matches.some(match => d.label.toLocaleLowerCase().includes(match)));
```

Comment:
Don't think `device` needs to be a `let` here.

Reply:
You're right, it doesn't. Fixed